app.js

```diff
@@ -0,0 +1,54 @@
+const form = document.getElementById('form');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+let cookie;
+let cookieArray;
+const bookInfo = document.getElementById('bookInfo');
+let bookArray = [];
+
+if (localStorage.getItem('bookStored') !== null) {
+  cookie = localStorage.getItem('bookStored');
+  cookieArray = JSON.parse(cookie);
+
+  if (cookieArray) {
+    bookArray = cookieArray;
+  }
+}
+function displayBooks() {
+  bookArray.forEach((item) => {
+    const newNode = document.createTextNode('');
+    const titleHtml = document.createElement('p');
+    const authorHtml = document.createElement('p');
+    const remove = document.createElement('button');
+    const indexVal = bookArray.indexOf(item);
+    const recoveredObject = JSON.parse(item);
+    titleHtml.innerText = recoveredObject.title;
+    authorHtml.innerText = recoveredObject.author;
+    remove.innerText = 'Remove!';
+    bookInfo.append(titleHtml, authorHtml, remove);
+
+    remove.addEventListener('click', () => {
+      bookArray = JSON.parse(localStorage.getItem('bookStored'));
+      bookArray.splice(indexVal, 1);
+      localStorage.setItem('bookStored', JSON.stringify(bookArray));
+      titleHtml.replaceWith(newNode);
+      authorHtml.replaceWith(newNode);
+      remove.remove();
+    });
+  });
+}
```

Comment:
Great job team 🤝

- [ ] [Optional] You may use ES6 syntax as this is a recommended way to write functions and methods instead of using the `function` keyword.
- [ ] [Optional] You may also have separate functions one to add a book and another to remove a book. This was suggested in the project requirements.